src/Services/LoggingService.php

```diff
@@ -113,6 +127,7 @@ public function extractVars($loggingInfo): array {
     $method = $loggingInfo['method'] ?? null;
     $referenceType = $loggingInfo['referenceType'] ?? null;
     $referenceValue = (int) $loggingInfo['referenceValue'] ?? null;
+    $additionalInfo[self::WAYFAIR_PLUGIN_VERSION] = $this->version;
```

Comment:
If $loggingInfo['additionalInfo'] was null on line 126 then this version information will not actually get set

Reply:
I am not sure why it's not showing up but on my PR it gives an empty array if $loggingInfo['additionalInfo'] was falsy on line 126

Reply:
I should have looked at the next line as I was assuming the intent was also to set the loggingInfo if it wasn't set.